### PR TITLE
Wishlistpage: refactor cards and footer

### PR DESCRIPTION
**Pull request by Ayushi20-19**
image

### CODE REVIEW

**Review by Divya70 — APPROVED** [review]

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    <!-- blossom component library -->
+    <link
+      rel="stylesheet"
+      href="https://blossom-ui.netlify.app/components/styles.css"
+    />
+
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="wishlist.css" />
+
+    <title>Document</title>
```
Comment (shaik-saad) [inline wishlist.html]: You can change the title to page name.

File: wishlist.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            <li>
+              <a href="#"><i class="fas fa-shopping-cart"></i></a>
+            </li>
+
+            <li>
+              <a href="#"><i class="fas fa-user-plus"></i></a>
+            </li>
+          </ul>
+        </nav>
+        <a class="ham-burger" onclick="sideBarShow()">
```
Comment (shaik-saad) [inline wishlist.html:44]: In levelzero, Tanay Bhaiya has mentioned to not use onClick() and rather use addEventListener in js.

**Review by shaik-saad — APPROVED** [review]
Added two suggestions. You're good to merge.

File: wishlist.html (hunk 1 above)
Comment (Ayushi20-19) [reply wishlist.html]: oh yes

**Pull request merged**